pkgs/top-level/all-packages.nix

```diff
@@ -2926,6 +2926,11 @@ in
 
   epubcheck = callPackage ../tools/text/epubcheck { };
 
+  luckybackup = callPackage ../tools/backup/luckybackup {
+    inherit (qt5) qmake;
```

Comment:
Please use `libsForQt5.callPackage` here. That makes it way easier to add further qt5 deps in the future if needed :)

Reply:
It still needs to declare qmake, right?